components/ExampleComponent/ExampleComponent.mdx

```diff
@@ -5,11 +5,11 @@ export const ExampleComponent = ({ header }) => {
         <p className="text-lg font-bold">{header}</p>
         <p>
           <i className="fa-solid fa-code mr-2" />
-          Write <a href="https://docs.readme.com/main/docs/building-custom-mdx-components" target="_blank">MDX</a> to render interactive content
+          Write <a href="https://docs.readme.com/main/docs/building-custom-mdx-components" target="_blank" rel="noreferrer">MDX</a> to render interactive content
```

Comment:
This came from [react/jsx-no-target-blank](https://github.com/jsx-eslint/eslint-plugin-react/tree/master/docs/rules/jsx-no-target-blank.md) which requires that `rel=noreferrer` always be present on links that are `target=_blank`.